acra/security-controls/searchable-encryption/_index.md

```diff
@@ -2,3 +2,38 @@
 title: Searchable encryption
 bookCollapseSection: true
 ---
+
+# Searchable encryption
+
+Search for database rows knowing exact value of their encrypted column.
+
+Searchable encryption makes it possible to have column whose value is encrypted, but nevertheless,
+you will be able to find it **if and only if you know its exact value**.
+Even the smallest mismatch between searched and encrypted values will give no results.
+That's because Acra is comparing SHA256-based HMACs of values under the hood.
```

Comment:
would be nice to mention that values may be sent from the app as acrastructs or as plaintext. In case when app sent acrastruct, AcraServer will decrypt ciphertext and calculate hmac, after that send it to the database.